data/mods/ssb/moves.js

```diff
@@ -1060,137 +1117,46 @@ let BattleMovedex = {
 		type: "Rock",
 	},
 	// Forrce
-	purplepills: {
-		accuracy: true,
-		basePower: 0,
-		category: "Status",
-		desc: "The user gains a random typing and 3 moves based on that typing (2 special moves and 1 status move). The user's attacks deal damage based off the user's Special Defense. If used again, returns the user to its original moveset and typing. This move fails if the user is not Forrce.",
-		shortDesc: "Forrce: Gains 3 random moves and typing.",
-		id: "purplepills",
-		name: "Purple Pills",
+	allduerespect: {
+		accuracy: 99,
+		basePower: 80,
+		category: "Special",
 		isNonstandard: "Custom",
-		pp: 15,
+		desc: "This attack is super effective if the foe has any attacks that are supper effective against the user. This move becomes a physical attack if the user's Attack is greater than its Special Attack, including stat stage changes.",
+		shortDesc: "SE if foe has SE moves. Physical if Atk > Sp. Atk.",
+		id: "allduerespect",
+		name: "All Due Respect",
+		pp: 5,
 		priority: 0,
-		flags: {},
+		flags: {protect: 1, mirror: 1},
 		onTryMove() {
 			this.attrLastMove('[still]');
 		},
 		onPrepareHit(target, source) {
-			this.add('-anim', source, "Swallow", source);
+			this.add('-anim', source, 'Fusion Flare', target);
 		},
-		onTryHit(target, source) {
-			if (source.name !== 'Forrce') {
-				this.add('-fail', source);
-				this.hint("Only Forrce can use Purple Pills.");
-				return null;
-			}
+		hasCustomRecoil: true,
+		onMoveFail(target, source, move) {
+			// Secret ;)
```

Comment:
No, were not keeping this secret. Put it in the move description.

Reply:
yeah i was going to, but eventually forgot to push. sorry